build.ps1

```diff
@@ -360,8 +366,14 @@ try
         # Override verbosity to display individual test execution.
         $oldVerbosity = $verbosity
         $verbosity = "normal"
+        $testCategory = "TestCategory=E2E"
+        
+        if ($skipDeviceStreamTests) 
```

Comment:
I would add a note here stating that we are intentionally skipping e2e tests for batched/CI builds for certian scenarios.